.tool-versions

```diff
@@ -1,17 +0,0 @@
-act 0.2.42
-actionlint 1.6.23
-bosh 7.1.3
-cf 8.6.0
-concourse 7.9.1
-direnv 2.32.2
-gcloud 418.0.0
-ginkgo 2.8.3
-golang 1.20.1
-golangci-lint 1.51.2
-java temurin-17.0.5+8
-make 4.4
-maven 3.8.6
-nodejs 18.14.1
-ruby 3.2.1
-shellcheck 0.9.0
-yq 4.30.8
```

Comment:
Removing this file could cause problems for our github-actions. Probably that can only be done later after the Nix-migration.